### PR TITLE
Let user specify the firebase app name

### PR DESCRIPTION
**Pull request by MvRemmerden**

### CODE REVIEW

**Comment by MvRemmerden** [comment]
@kmjennison I think this PR might be ready, would you mind approving it to be deployed on Vercel so we can see whether any of the checks fail? 🙂

**Review by kmjennison — CHANGES_REQUESTED** [review]
@MvRemmerden Thanks for this! The approach looks good. Could you investigate the failing tests?

**Comment by MvRemmerden** [comment]
@kmjennison I might need some help here. It looks like the problem is due to this change here: https://github.com/gladly-team/next-firebase-auth/pull/716/files#diff-40051885c173eb84152e58b7e9f3cbac7bd1e5d9de2520a1bd2a4253b4a6b9a0R108

I'm not sure if we just need to adapt the test to ensure that the app is initialized before, or if there is a bug here in the actual code.

Would you mind having a look yourself?

**Comment by MvRemmerden** [comment]
Update: I think I got it working, it might have just been missing mocked configs in the tests, everything is passing now ✅

File: src/configTypes.ts [hunk 1]
```diff
@@ -110,6 +111,7 @@ export const defaultConfig = {
   onVerifyTokenError: (_err: Error) => {},
   // eslint-disable-next-line @typescript-eslint/no-empty-function, @typescript-eslint/no-unused-vars
   onTokenRefreshError: (_err: Error) => {},
+  firebaseClientAppName: 'example-app-name',
```
Comment (kmjennison) [inline src/configTypes.ts]:
```suggestion
```

We shouldn't set a default app name.

**Review by kmjennison — CHANGES_REQUESTED** [review]
One change needed and otherwise looks good.

**Comment by MvRemmerden** [comment]
@kmjennison Awesome, committed, back to you 🙂

**Review by kmjennison — APPROVED** [review]
LGTM! Thanks for your work on this!

**Pull request merged**